src/stylus/CargoStylusFactory.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+interface IArbWasm {
+    function activateProgram(
+        address program
+    ) external payable returns (uint16 version, uint256 dataFee);
+}
+
+contract CargoStylusFactory {
```

Comment:
missing a payable receive function to receive refund from constructor (if any)

Reply:
specifically, I mean if `dataFee < _activationValue` in
```
(, uint256 dataFee) = IArbWasm(ARB_WASM).activateProgram{value: _activationValue}(
            _contract
        );
```
I am not sure if that refund involves evm call tho